### ISSUE TITLE
Switch to USGS base map after zoom level ~3?

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/alaska-wildfires#112]
The motivation here is that at local scales, users might be better able to orient self/data against the cartographic product(s) offered by the USGS National Map.  What would this look like?  Is it better?

**Comment by cstephen** [comment]
I experimented with this a bit, and unfortunately the USGS National Map doesn't play nice with the EPSG:3338 projection.

Here's what the USGS National Map looks like in EPSG:3857 (like we use for Northern Climate Reports mini-maps), centered on Fairbanks:

![Image](https://github.com/user-attachments/assets/d7c07dff-4dc5-4af6-b0c8-d9cf231481d0)

And here's what the same USGS National Map layer looks like in EPSG:3338, centered on Fairbanks:

![Image](https://github.com/user-attachments/assets/a95f59f0-5b74-4823-b85f-aa29948ae33c)

Notice how tiny the labels are in the second screenshot. Unfortunately they stay that tiny regardless of how far you zoom in.

I think it's probably a bad idea to try to switch the map projection when the user zooms in (for a number of reasons, not to mention the work involved), so I think this is a dead end unfortunately.

I also experimented with displaying the Natural Earth "Populated Places" layer overtop the map to give more spatial context, but this often adds a place label where there had already been one on the OSM layer. The OSM layer already does a pretty good job of showing additional place labels for smaller communities as you zoom farther into the map, so I don't think there's a great way to add additional place labels via a (population filtered) "Populate Places" layer without adding duplicate labels in many places.

Long story short, I think the OSM layer already does a pretty decent job of adding spatial context to the map via place labels. It's also a very clean layer with less visual distractions to compete with the various data layers (vs. the USGS National Map). So, I'm going to call this investigation complete and close this issue.

**Comment by cstephen** [comment]
Re-opened this issue. Let's take another look at this, but change the map projection as needed at higher zoom levels to fix the tiny-label USGS National Map issue mentioned above.

**Comment by brucecrevensten** [comment]
We won't move forward here due to the renaming of map elements on the USGS National Map.
